src/jit/InstList.cpp

```diff
@@ -314,16 +314,18 @@ void JITCompiler::insertStackInitList(InstructionListItem* prev, size_t variable
     }
 }
 
+const char* JITCompiler::m_byteCodeNames[] = {
```

Comment:
`m_byteCodeNames` array is used only for dump.
For those debugging tools, what about dynamically allocating this array only when dump operation is initialized?

Reply:
I am not sure how can I do that. To do this dynamically, I need some static information, which can be used to generate this array. Shall we use some external file, which is processed at runtime? Or some kind of compressed data structure, which is uncompressed at runtime?

Reply:
Can we enable all dump operations only for debug build?

Reply:
Yes, good idea. Now the byte code dump can be disabled even if perf dump is enabled.